salt/output/highstate.py

```diff
@@ -221,11 +223,14 @@ def _format_host(host, data):
                               .format(ret.get('duration', 0)))
 
             tcolor = colors['GREEN']
-            orchestration = ret.get('__orchestration__', False)
-            schanged, ctext = _format_changes(ret['changes'], orchestration)
-            if not ctext and 'pchanges' in ret:
-                schanged, ctext = _format_changes(ret['pchanges'], orchestration)
-            nchanges += 1 if schanged else 0
+            if ret.get('name') in ['state.orch', 'state.orchestrate', 'state.sls']:
+                nested = output(ret['changes']['return'], indent_level=indent_level+1)
+                ctext = re.sub('^',u' ' * 14 * indent_level, '\n'+nested, flags=re.MULTILINE)
```

Comment:
We shouldn't be using the `u''` syntax here. That should be using `salt.utils.stringutils.to_unicode` instead.

Reply:
Ah my bad, originally wrote this on a release branch pre Unicode change. Will update and add a regression test

Reply:
@mattp- All good! Thank you :)

Reply:
The `u` is still there. Also, there need to be spaces after commas and between operators, or else the linter will complain.

I don't think we'll need to use `salt.utils.stringutils.to_unicode()` here, due to `unicode_literals` being imported. The `nested` variable should contain a unicode string because the `output()` function returns unicode strings.